test/units/vars/test_variable_manager.py

```diff
@@ -21,8 +21,6 @@
 
 import os
 
-from collections import defaultdict
```

Comment:
I could remove the changes to this file, it's irrelevant to the feature I've added. I just noticed it was masking the real fact_cache and so the tests weren't failing when I broke it.

Reply:
we kind of mask it in variablemanager itself, but we should not in these tests